inlong-sort/sort-connectors/pom.xml

```diff
@@ -59,6 +59,11 @@
             <artifactId>flink-streaming-java_${flink.scala.binary.version}</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-table-api-java-bridge_2.11</artifactId>
```

Comment:
It is recommended to use ${flink.scala.binary.version} instead of the scala version number